docs/topics/mta-web-config-maven-repo-size.adoc

```diff
@@ -8,6 +8,11 @@
 
 You can reduce the size of a Maven repository in the *Repositories* view of the {ProductName} ({ProductShortName}) {WebName}.
 
+[NOTE]
+====
+This procedure is possible only if the `rwx_supported` configuration option of the Tackle CR is set to `true`. Otherwise, both the *Local artifact repository* field and the *Clear repository* button are disabled.
+====
```

Comment:
This is NOT correct. `rwx_supported` parameter can be missed and absent in the CR and still field and button will be ENABLED. 
To disable them - `false` value should be set.

Reply:
@ibragins 
Corrected, please take a look.